packages/client/components/SideBarStartMeetingButton.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {useLocation} from 'react-router'
+import useRouter from '~/hooks/useRouter'
+import getTeamIdFromPathname from '~/utils/getTeamIdFromPathname'
+import {BezierCurve} from '../types/constEnums'
+import FlatPrimaryButton from './FlatPrimaryButton'
+import Icon from './Icon'
+
+const Button = styled(FlatPrimaryButton)<{isOpen: boolean}>(({isOpen}) => ({
+  height: 50,
+  overflow: 'hidden',
+  padding: 0,
+  width: isOpen ? 160 : 50,
+  marginLeft: 7,
+  marginTop: 15,
+  marginBottom: 15,
+  transition: `all 300ms ${BezierCurve.DECELERATE}`,
+  justifyContent: 'flex-start'
+}))
+
+const MeetingIcon = styled(Icon)({
+  margin: '0px 12px',
```

Comment:
+1 setting this to '0px 11px' seems centered to me with 48px width.

Reply:
ok

Reply:
@Dschoordsch 
If you know anything about this, I am started getting errors on this branch in `NewMeetingCarousel`
```
export 'Swiper' (imported as 'Swiper') was not found in 'swiper/react' (module has no exports)
```

Reply:
Sounds like you pulled a new version and did not run `yarn` again to install the package?

Reply:
Started working after restart👍